src/main/java/org/fundacionjala/core/util/Environment.java

```diff
@@ -35,6 +35,7 @@ private void readFile() {
             properties.load(input);
         } catch (IOException ex) {
             LOGGER.error(ex.getMessage());
+            //TODO stop execution with message
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)